client/src/game/ui/reactive/view/cardsView.ts

```diff
@@ -110,6 +110,23 @@ const subscribeToCardChanges = (order: number) => {
     location: c.location,
   }), () => updateBorder(order));
 
+  // Card fade and critical indicator
+  subFullState((s) => {
+    const card = s.visibleState!.deck[order];
+    const status = card.suitIndex !== null && card.rank !== null
```

Comment:
parens around the condition because the operator precedence of ? is not clear

Reply:
if it's the other grouping than we're &&-ing with a non boolean. I don't think that's possible

Reply:
You can definitely && a non-boolean - welcome to javascript :/ Everything casts to everything silently

Reply:
fair. fixed